.github/workflows/dependabot-automerge.yaml

```diff
@@ -0,0 +1,21 @@
+# Automatically merge Dependabot PRs upon approval by leaving
+# a comment on Dependabot's pull-request.
+
+name: Automerge Dependabot PR
+on:
+  pull_request_review:
+    types: [submitted]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  comment:
+    if: ${{ github.event.review.state == 'approved' && github.event.pull_request.user.login == 'dependabot[bot]' }}
```

Comment:
Is it possible to set the git user to `dependabot[bot]` and bypass this check in a PR?

Reply:
What do you mean?

I wanted this workflow to run for PRs opened _only_ by Dependabot, but I couldn’t find a better way to check that than
```yaml
github.event.pull_request.user.login == 'dependabot[bot]'
```

Reply:
I was wondering if it's possible to impersonate `dependabot[bot]`. If it's not possible, we can assume this check works.

Reply:
We tried it, can’t be impersonated by changing my local `git.name` settings.